test/fixtures/reports.yml

```
@@ -246,3 +246,35 @@ report30:
   description: |-
     頑張れませんでした
   reported_on: <%= Time.now - 1.month - 1.day %>
+
+report31:
+  user: kananashi
+  title: 最新のwipの日報
+  description: |-
+    最新のwipの日報です
+  reported_on: <%= Time.now %>
+  wip: true
+
+report32:
+  user: kananashi
+  title: wipを除いた最新の日報
+  description: |-
+    wipを除いた最新の日報です
+  reported_on: <%= Time.now - 1.day %>
+
+report33:
+  user: kananashi
+  title: 9日ぶりの日報
+  emotion: 1
+  description: |-
+    最新から9日ぶりの日報でwipにしています
+    今日の気分がなしでwipから投稿ができないので、emotionを設定
+  reported_on: <%= Time.now - 10.day %>
+  wip: true
+
+report34:
+  user: kananashi
+  title: 10ぶりの日報
+  description: |-
+    最新から10ぶりの日報です
```

[要確認]こちら、単位の「日」を追記ください〜

```suggestion
  title: 10日ぶりの日報
  description: |-
    最新から10日ぶりの日報です
```